bin/hsw-cli

```diff
@@ -592,93 +592,76 @@ class CLI {
+        this.log('  $ coins: View wallet coins.');
+        this.log('  $ dump [address]: Get wallet key WIF by address.');
+        this.log('  $ get: View wallet.');
+        this.log('  $ history: View TX history.');
+        this.log('  $ import [wif|hex]: Import private or public key.');
+        this.log('  $ key [address]: Get wallet key by address.');
+        this.log('  $ listen: Listen for events.');
+        this.log('  $ lock: Lock wallet.');
+        this.log('  $ master: View wallet master key.');
+        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ pending: View pending TXs.');
+        this.log('  $ rescan [height]: Rescan for transactions.');
         this.log('  $ retoken: Create new api key.');
         this.log('  $ send [address] [value]: Send transaction.');
-        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ shared add [xpubkey]: Add key to wallet.');
+        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
         this.log('  $ sign [tx-hex]: Sign transaction.');
-        this.log('  $ zap [age?]: Zap pending wallet TXs.');
-        this.log('  $ abandon [hash]: Abandon a transaction.');
         this.log('  $ tx [hash]: View transaction details.');
-        this.log('  $ blocks: List wallet blocks.');
-        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
         this.log('  $ view [tx-hex]: Parse and view transaction.');
-        this.log('  $ import [wif|hex]: Import private or public key.');
         this.log('  $ watch [address]: Import an address.');
-        this.log('  $ key [address]: Get wallet key by address.');
-        this.log('  $ dump [address]: Get wallet key WIF by address.');
-        this.log('  $ lock: Lock wallet.');
-        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
+        this.log('  $ zap [age?]: Zap pending wallet TXs.');
+        this.log('');
+        this.log('Admin commands, they require api-token if node is run with wallet-wallet-auth flag.');
+        this.log('Otherwise they can be called as the above ones, without any additional parameter.');
+        this.log('  $ backup [path]: Backup the wallet db.');
+        this.log('  $ mkwallet [id]: Create wallet.');
+        this.log('  $ rpc [command] [args]: Execute RPC command.');
         this.log('  $ resend: Resend pending transactions.');
-        this.log('  $ rescan [height]: Rescan for transactions.');
-        this.log('Other Options:');
+        this.log('  $ wallets: List all wallets.');
+        this.log('');
+        this.log('Other options:');
+        this.log('  --id [wallet id]: Wallet id.');
         this.log('  --passphrase [passphrase]: For signing/account-creation.');
         this.log('  --account [account-name]: Account name.');
+        this.log('  --api-token [token]: API or Admin token to run commands');
```

Comment:
Same as above. This is just `--token [32 byte hex string]: Admin or Wallet token when walletAuth is true.`

Reply:
Do we want to mention basic configs here?
  - Wallet http server configs:
    - `--http-host`, `--http-port`(or alternatively `--url`)
    - `--ssl`
    - `--api-key` - Basic auth for the wallet http server
  - Wallet node configs
    - `--network`?

Reply:
Added `--api-key`, but not the rest.
I think we need to decide, if we want the hint to be tiny or to list all the commands.
I'm for listing all the parameters (and api commands), however it will require more things to add there.